### ISSUE TITLE
Hamburger button does not work 

### ISSUE THREAD

**Issue by danielmrey** [BST-Plus/BST-Plus#9]
I downloaded the code from here: https://github.com/BST-Plus/BST-Plus and the Hamburger button does not work, but it does if I download the code from here: https://github.com/BST-Plus/BST-Plus/releases > v3

Thanks

**Comment by SimonPadbury** [comment]
Thanks. I will look into it tomorrow. 
Can you see what's wrong with it?

**Comment by danielmrey** [comment]
Simply that the Hamburger button does not work

Thanks

**Comment by SimonPadbury** [comment]
Both hamburgers on BST-Plus are working for me - on a fresh download from this repository.

Sorry, what is the problem?

**Comment by danielmrey** [comment]
It happened in two occasions when I downloaded the code from here:
https://github.com/BST-Plus/BST-Plus

Hence I decided to use the code in v3 instead:
https://github.com/BST-Plus/BST-Plus/releases

But if you have tried and you say it works, then it must work then...

Thank you for taking your time!

daniel.

Thank you.

Best Regards.

_Daniel Muñoz Rey_
Web designer and developer
www.danielmrey.com
info@danielmrey.com
linkedin.com/in/danielmrey https://www.linkedin.com/in/danielmrey
+44 (0)776 517 0430
